src/test/java/net/sf/jabref/logic/importer/fetcher/GvkFetcherTest.java

```diff
@@ -0,0 +1,122 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.BibLatexEntryTypes;
+import net.sf.jabref.preferences.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class GvkFetcherTest {
+
+    private GvkFetcher fetcher;
+    private BibEntry bibEntryPPN591166003;
+    private BibEntry bibEntryPPN66391437X;
+
+    @Before
+    public void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
```

Comment:
Do you really need this? Most of the Globals dependencies are removed now, could be worthwhile trying without it.